ev-dev-tools/src/ev_cli/ev.py

```diff
@@ -617,15 +630,17 @@ def types_genhdr(args):
 
 
 def main():
-    global validators, everest_dir
+    global validators, everest_dirs, work_dir
 
     parser = argparse.ArgumentParser(description='Everest command line tool')
     parser.add_argument('--version', action='version', version=f'%(prog)s {__version__}')
 
     common_parser = argparse.ArgumentParser(add_help=False)
 
-    common_parser.add_argument("--everest-dir", "-ed", type=str,
-                               help='everest directory containing the interface definitions (default: .)', default=str(Path.cwd()))
+    common_parser.add_argument("--work-dir", "-wd", type=str,
+                               help='work directory containing the manifest definitions (default: .)', default=str(Path.cwd()))
+    common_parser.add_argument("--everest-dir", "-ed", nargs='*',
```

Comment:
Problem with nargs here is that ``ev-cli mod c --schemas-dir /workspace/everest-framework/schemas --everest-dir /workspace/everest-core Fool`` won't work, because the last argument gets digested by everest-dir instead of being module name.  This can be worked around by putting everest-dir in front of schemas-dir, but what if schemas-dir is not used? I don't know a proper solution right now, but as far as I remember using optional arguments with nargs is not good practice